### ISSUE TITLE
compilation fails when using epoxy

### ISSUE THREAD

**Issue by stkw0** [gabime/spdlog#391]
When using epoxy and spdlog at the same time it fails to compile with the next error

```
In file included from /usr/include/epoxy/glx.h:36:0,
                 from a.cpp:1:
/usr/include/spdlog/fmt/bundled/format.h:1431:8: error: expected identifier before numeric constant
 struct False
        ^
/usr/include/spdlog/fmt/bundled/format.h:1431:8: error: expected unqualified-id before numeric constant
In file included from /usr/include/spdlog/fmt/fmt.h:21:0,
                 from /usr/include/spdlog/common.h:41,
                 from /usr/include/spdlog/spdlog.h:13,
                 from a.cpp:2:
/usr/include/spdlog/fmt/bundled/format.h: En la función ‘void fmt::internal::format_arg(Formatter&, const Char*, const T&)’:
/usr/include/spdlog/fmt/bundled/format.h:1484:5: error: expected primary-expression before ‘>’ token
     FMT_STATIC_ASSERT(False<T>::value,
     ^
/usr/include/spdlog/fmt/bundled/format.h:1484:5: error: ‘::value’ no se ha declarado
     FMT_STATIC_ASSERT(False<T>::value,
     ^
```

The full source is the next:
```cpp
#include <epoxy/glx.h>
#include <spdlog/spdlog.h>

int main() {
}
```

I used gcc version 5.4.0.
I have tested the next spdlog versions: 0.11.0 0.12.0 and the git from master branch.
With spdlog version 0.10.0 it don't fails. 

This error seems to come from the fmt library used by spdlog. Replacing the bundled version of fmt library  with the latest libfmt version (3.0.1) solves the problem.

Probably we should update the bundled version of fmt

Thank you.

**Comment by victoredwardocallaghan** [comment]
@stkw0 we resolved the issue in 7fa8f8f @gabime any chance of a bug fix release soon?

**Comment by gabime** [comment]
Merged in 43BDQC

**Comment by victoredwardocallaghan** [comment]
@stkw0 @gabime unfortunately we can't just  use a live ebuild of spdlog to workaround our issue as it brings in all the build dependencies of spdlog into our production environment. Since commit e8da69e is essentially a critical fix could we request a bugfix release be done to resolve our issue? Without the fix spdlog is essentially completely broken for us after version 0.10.0.

**Comment by stkw0** [comment]
As the fix got merged I suppose that I can close the issue

@victoredwardocallaghan If it's a different issue or an issue related to an specific distro I don't think this is the best place to discuss it. btw, you CAN use the live ebuild without the mentioned problem. If you have some problem with this go to the linux distribution forum/IRC/etc.

**Comment by gabime** [comment]
@victoredwardocallaghan I just released version 0.13.0 to cover this

**Comment by victoredwardocallaghan** [comment]
@gabime thank you !
